exotic/plots.py

```diff
@@ -140,19 +140,19 @@ def plot_flux(times, targ, targ_unc, ref, ref_unc, norm_flux, norm_unc, airmass,
 def plot_variable_residuals(save):
     plt.title("Stellar Variability Residuals")
     plt.ylabel("Residuals (flux)")
-    plt.xlabel("Time (BJD_TDB)")
+    plt.xlabel("Time (JD)")
```

Comment:
Isn't this still in BJD_TDB? We do the conversions ourselves to BJD_TDB, so I think this label is still accurate.

Reply:
AAVSO requires us to save the it in the JD time format. I saved the corresponding JD times to the BJD ones. For reference, here is [AAVSO's](https://www.aavso.org/aavso-extended-file-format) available time formats:

> - **DATE**: The format of the date used in the report. Times are midpoint of the observation. Convert all times from UT to one of the following formats:
> 1. JD: Julian Day (Ex: 2454101.7563)
> 2. HJD: Heliocentric Julian Day
> 3. EXCEL: the format created by Excel's NOW() function (Ex: 12/31/2007 12:59:59 a.m )